### ISSUE TITLE
False positives in prefer-const rule

### ISSUE THREAD

**Issue by ota-meshi** [ota-meshi/svelte-eslint-parser#9]
The following code will falsely report it.

```svelte
<script>
    let a = 'abc';
</script>
<input bind:value={a}>
```

**Comment by ota-meshi** [comment]
fixed #12
